### ISSUE TITLE
Doorbell Dual Package Detection

### ISSUE THREAD

**Issue by ejpenney** [fuatakgun/eufy_security#399]
Eufy Doorbell dual supports package detection but no sensor for this exists in the Eufy Security component.

It would be really cool if we had a binary sensor that turned on when a package was detected.

I've been trying to use IFTTT to capture Android Package notifications and send web requests, but that's been unreliable, to say the least.

Glancing through [eufy-security-client](https://github.com/bropat/eufy-security-client/blame/54f94083592feeeaf8e53ab77b521f4cd62aea04/src/p2p/types.ts#L471) it looks like some support for turning on and off package guard has been added.  I'm not seeing anything related to package detection, but I'm just searching the source for the word "package".

**Comment by fuatakgun** [comment]
You should be able to see the whole device attributes under entity state attributes. Can you confirm if any attribute changes when you drop a package after camera detection.

After your validation, i can expose the state attribute as a binary sensor.

**Comment by ejpenney** [comment]
I'm not seeing anything but I'm not sure I'm looking in the right place?  Do you mean the primary entity?  In my case, alarm_cotrol_panel.doorbell?

I just realized I mixed up my two tickets as well.  I've asked upstream if _this_ is supported [here](https://github.com/fuatakgun/eufy_security/issues/400)  I'll update my links for the other question...

**Comment by ejpenney** [comment]
He marked this completed with the 1.0.0 release, but I've noticed just pulling the 1.0.0 container isn't enough to enable this.  I'm not seeing anything related to packages in the main device's attributes.  Haven't tried with 1.0.1 or 1.0.2.

Sounds like there are other compatibility issues on this side of the fence that need to be worked on, correct?

**Comment by ejpenney** [comment]
This is the ticket I meant it for:

How's this work? I'm running add-on 1.3.0 and component 2.10.0 and don't see the package status.  Is there supposed to be a binary_sensor, or should I look for an event?

**Comment by fuatakgun** [comment]
You need to move into alpha release and share your debug sensor attributes for device so i can take a look at fields.

Furthermore, you can share the device with me following readme file and i can debug it.

**Comment by fuatakgun** [comment]
delivered in ALPHA version (v5.x.x)

**Comment by fuatakgun** [comment]
do you have any missing sensor that you would like to have?

**Comment by ejpenney** [comment]
Yes, I would like a sensor indicating a package is currently detected.

**Comment by fuatakgun** [comment]
don't you have already `packageDelivered` sensor?

**Comment by fuatakgun** [comment]
Or, anything you see in your device state, I can expose it as a sensor :) please check what you have and what not and ask from that list

**Comment by ejpenney** [comment]
Well, I did... I just setup a template sensor to read from the debug sensor, as an experiment.  Was fooling around trying to get the sensor to trip, unsuccessfully, seems like it's not updating?  Anyways, I updated to 6.1.0 and the packageDelivered property disappeared.

```
properties:
  name: Doorbell
  model: T8203
  serialNumber: T8203P1122020012
  hardwareVersion: P1
  softwareVersion: 2.0.6.6
  type: 93
  stationSerialNumber: T8203P1122020012
  wifiRssi: -39
  wifiSignalLevel: 4
  enabled: true
  autoNightvision: true
  statusLed: true
  motionDetection: true
  motionDetectionType: 0
  motionDetectionSensitivity: 4
  motionDetected: false
  personDetected: false
  personName: ''
  watermark: 1
  pictureUrl: REDACTED
  state: 1
  ringing: false
  speakerVolume: 80
  ringtoneVolume: 80
  audioRecording: true
  videoStreamingQuality: 1
  videoWdr: true
  notificationType: 2
  notificationRing: true
  notificationMotion: true
  deliveryGuard: true
  deliveryGuardPackageGuarding: true
  deliveryGuardPackageGuardingVoiceResponseVoice: 1
  deliveryGuardPackageGuardingActivatedTimeFrom: '00:00'
  deliveryGuardPackageGuardingActivatedTimeTo: '23:59'
  deliveryGuardUncollectedPackageAlert: true
  deliveryGuardUncollectedPackageAlertTimeToCheck: '20:00'
  deliveryGuardPackageLiveCheckAssistance: true
  dualCamWatchViewMode: 12
  ringAutoResponse: false
  ringAutoResponseVoiceResponse: false
  ringAutoResponseVoiceResponseVoice: 1100476313
  ringAutoResponseTimeFrom: '00:00'
  ringAutoResponseTimeTo: '23:59'
  snooze: false
  snoozeTime: 0
  snoozeStartTime: 0
  snoozeHomebase: false
  snoozeMotion: false
  snoozeChime: false
```

**Comment by ejpenney** [comment]
I just reverted to 6.0.0 and the packageDelivered property came back.

**Comment by ejpenney** [comment]
I don't think this is related to 6.0 vs 6.1.  I can't get packageDelivered to switch to True, I tried laying down a package, got a notification in the Eufy app that a package had been delivered, but the property remained False.  I restarted the Add-On and no change, I reload the integration and it requires a reboot.  When I rebooted the property disappeared again.  So it seems the property comes and goes.
